source_test.yaml

```
@@ -264,6 +264,21 @@
   editable: False
   strict_validation: True
 
+- name: 'openeuler'
+  versions_from_repo: False
+  rest_api_url: 'https://repo.openeuler.org/security/data/osv/all.json'
+  type: 2
+  ignore_patterns: ['^(?!OESA-).*$']
+  directory_path: 'data/osv'
```

```suggestion
  directory_path: 'security/data/osv'
```

(This doesn't actually matter, the rest importer uses link and rest_api_url, but it's nice to be consistent)

---

Thanks for reviewing the code! We will update the prod PR to include this point as well.